service-tests/ansible.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'ansible', title: 'Ansible Galaxy' });
+module.exports = t;
+
+t.create('ansible role name')
+ .get('/role/14542.json')
+ .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('role'),
+    value: Joi.equal('openwisp.openwisp2')
+ }));
+
+t.create('ansible role download counts')
+ .get('/role-downloads/14542.json')
```

Comment:
Is there any precedent to this URL? Perhaps `/role/d` is better; that is more consistent with other badges.

Reply:
`/role/d` already existed before, therefore I thought it natural to avoid changing something people may be using to keep backward compatibility.

Reply:
PS: https://github.com/badges/shields/blob/master/server.js#L4449

Reply:
Fair enough, though `/role/d/14542.svg` wouldn't match `/^\/ansible\/(role)\/([^\/]+)\.(svg|png|gif|jpg|json)$/` because it has too many slashes.

Plus, you could restrict the role ID pattern to digits, it seems.

Reply:
ok sorry I don't know why I assumed that `d` stood for `(d+)` in regexp terms.

Reply:
I'm unsure if I need to improve something regarding this point or not. Should I use something like `/role/d/14542.svg`?

Reply:
Yes, please do!

Reply:
@paulmelnikow I don't know how to do this without complicating the code unnecessarily, eg: duplicating the `camp.route` definition or using an obscure regexp.

I spent several hours on this patch, I've added a test for code that I didn't write, I'm not even familiar with nodejs.
I'm sorry but I can't do better than this. I hope this patch will help to add this feature.

Reply:
Okay, thanks. I appreciate your directness. I'll give it a shot and see if I can get that added, and if not will merge like this.

Reply:
Done in https://github.com/paulmelnikow/shields/commit/f05de0aab58ce34a56f6b62b5dd1548e258304b7. I can't push to this PR, so I'll merge off of my branch.